### PR TITLE
Meal Backend

### PR DESCRIPTION
**Pull request by M7Mansour**
build: created route and controller for meal endpoint

### CODE REVIEW

File: server/src/routes/index.js [hunk 1]
```diff
@@ -1,9 +1,11 @@
 const router = require('express').Router();
 const mealsRouter = require('./meals');
 const categoryRouter = require('./categories');
+const mealRouter = require('./meal');
 const restaurantRouter = require('./restaurants');
 
 router.use('/meals', mealsRouter);
+router.use('/meal', mealRouter);
```
[the remark this answers is not in the document]
Comment (M7Mansour) [reply server/src/routes/index.js]: done

**Review by NourAdawi — APPROVED** [review]

**Review by Ahmad-Omar — APPROVED** [review]

File: server/src/config/dbBuild.js [hunk 2]
```diff
@@ -11,20 +11,27 @@ const { sequelize } = require('./connection');
...
+    await Promise.all(data.meals.map((item) => Meal.create(item)));
+    await Promise.all(
+      data.discountCode.map((item) => DiscountCode.create(item))
+    );
 
-  console.log('Database filled with data!');
+    console.log('Database filled with data!');
+  } catch (err) {
+    console.log(err);
+    console.log('hhhhhhhhhhhhhhhhhhhhhhhhhhh');
```
Comment (Amoodaa) [inline server/src/config/dbBuild.js:33]: whats happening here

**Review by Amoodaa — APPROVED** [review]

File: server/src/config/dbBuild.js (hunk 2 above)
Comment (M7Mansour) [reply server/src/config/dbBuild.js:33]: I was debuging and forgot to remove it 🤣

**Comment by Amoodaa** [comment]
resolve conflict

**Pull request merged**